zcutil/build.sh

```diff
@@ -39,10 +62,9 @@ then
     shift
 fi
 
-# BUG: parameterize the platform/host directory:
-PREFIX="$(pwd)/depends/x86_64-unknown-linux-gnu/"
+PREFIX="$(pwd)/depends/$BUILD/"
```

Comment:
I'm not sure off-hand whether this is sufficient to resolve the underlying bug, or whether there is more to do to get the dependencies to build on other platforms. But non-blocker because it resolves the bug stated in the file.